# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openh323/header-fix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur openh323_v1_18_0-orig/include/ixjlid.h openh323_v1_18_0/include/ixjlid.h
--- openh323_v1_18_0-orig/include/ixjlid.h	2008-03-14 09:32:08.000000000 +0100
+++ openh323_v1_18_0/include/ixjlid.h	2008-03-14 09:32:54.000000000 +0100
@@ -271,7 +271,6 @@
 
 #ifdef P_LINUX
 #include <linux/telephony.h>
-#include <linux/compiler.h>
 #include <linux/ixjuser.h>
 #endif
 
